wIndex/SettingsCache.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Interject Settings and Cache
+filename: "SettingsCache.md"
+layout: custom
+keywords: [Diagnostics, Tools, Settings, Configurations, Config]
+headings: ["Overview", "Opening the Folders", "Resetting Settings and Cache"]
+links: ["/wGetStarted/INTERJECT-Ribbon-Menu-Items.html#advanced-menu-items"]
+image_dir: "SettingsCache"
+images: [
+    {file: "DiagnosticsRibbon", type: "png", site: "Addin", cat: "Ribbon", sub: "", report: "", ribbon: "Advanced", config: ""}, 
+    {file: "Diagnostics", type: "png", site: "Addin", cat: "Diagnostics", sub: "Open User Folders", report: "", ribbon: "", config: ""}, 
+    {file: "ResetFileCache", type: "png", site: "Addin", cat: "Diagnostics", sub: "Reset File Cache", report: "", ribbon: "", config: ""}, 
+    {file: "ResetSettingsCache", type: "png", site: "Addin", cat: "Diagnostics", sub: "Reset Settings Cache", report: "", ribbon: "", config: ""}
+    ]
+description: Interject keeps multiple folders on your local drive for settings and a file cache. These folders help facilitate logging in, better performance, and functionality.
+---
+* * * 
+
+## Overview
+
+Interject keeps multiple folders on your local drive for settings and a file cache. These folders help facilitate logging in, better performance, and functionality.
+
+**File Cache** (stores cached reports for easy retrieval via the Report Library):
+
+- AppData/Local/Interject/FileCache
+
+**Settings** (stores cached settings, settings/configurations file, error logs, activity logging information, and login tokens):
+
+- AppData/Local/Interject/Settings
+- AppData/Roaming/Interject/Settings
+
+<br>
+
+<blockquote class=highlight_note>
+<b>Note:</b> Prior to version 2.5.1.1, Interject used the local folder for settings. Starting with 2.5.1.1, however, Interject stores the settings in the roaming profile so that settings can be synced across systems. The local folder still exists for archival purposes.
```

Comment:
small thing, but if users are still using (or can still use) 2.5.1.1, then the first sentence should be in present tense "interject uses..." because it's still happening or has the possibility of happening